packages/harvester-csi-driver/package.yaml

```diff
@@ -0,0 +1,3 @@
+url: https://github.com/harvester/charts.git
+subdirectory: charts/harvester-csi-driver
+commit: 40656f4e5024041a668b1c847ccb69250c3fd017
```

Comment:
Need to add version: https://github.com/rancher/charts/blob/33db4185684ece7b2ca1575cdc66af4841967764/packages/longhorn/package.yaml#L4

Reply:
No need to change the Chart.yaml; scripts will take care of that automatically. Just need to add this flag to package.yaml

Reply:
I find that the generated chart version is 100.0.0+up0.1.1, rather than 0.1.1. If I want to add it in the rancher kontainer-driver-metadata, which version is appropriate, 100.0.0+up0.1.1 or 0.1.1?